nodebuilder/daser/module.go

```diff
@@ -28,6 +28,7 @@ func Module(tp node.Type) fx.Option {
 					return das.Stop(ctx)
 				}),
 			)),
+			withMetrics(true), // TODO: make configurable when daser gets its own cfg
```

Comment:
Remove this line, should be disabled by default, and I will add to `nodebuilder/settings.go` once merged. The only reason this is difficult in this PR is because the node type isn't passed to `settings.go:WithMetrics` in main yet

Reply:
#1083 is an example of how it will look in the DI (I will be removing my daser things from there once this is merged)

Reply:
We can derive that the fact that metrics is on by checking if metrics endpoint is passed. Why would you prefer to move it outside of DASer pkg if we can pass in metricsEnabled flag with daser config instead?

Reply:
I got an idea. Sorry, thinking slow today. Moved option to `nodebuilder/settings.go` so it is consistent with other pkgs.